server/chat-plugins/othermetas.ts

```diff
@@ -542,4 +542,57 @@ export const commands: ChatCommands = {
 	crossevolvehelp: [
 		"/crossevo <base pokemon>, <evolved pokemon> - Shows the type and stats for the Cross Evolved Pokemon.",
 	],
+
+	'!showevo': true,
+	showevo(target) {
+		if (!this.runBroadcast()) return;
+		const targetid = toID(target);
+		if (!targetid) return this.parse('/help showevo');
+		const evo = Dex.getSpecies(target);
+		if (!evo.exists) {
+			return this.errorReply(`Error: Pokemon not found.`);
+		}
+		if (!evo.prevo) {
+			return this.errorReply(`Error: Pokemon does not evolve from another Pokemon.`);
+		}
+		const banlist = Dex.getFormat('gen8crossevolution').restricted || [];
+		if (banlist.includes(evo.name)) {
+			this.errorReply(`Warning: ${evo.name} is banned from being Cross Evolved into.`);
+		}
+		// Fake Pokemon
+		if (evo.isNonstandard === 'CAP') {
+			this.errorReply(`Warning: ${evo.name} is a fake Pokemon created by the CAP Project and is restricted to CAP.`);
+		}
+		const prevoSpecies = Dex.getSpecies(evo.prevo);
+		const deltas = Dex.deepClone(evo);
+		deltas.tier = 'CE';
+		deltas.weightkg = evo.weightkg - prevoSpecies.weightkg;
+		let i: StatName;
+		for (i in evo.baseStats) {
+			const statChange = evo.baseStats[i] - prevoSpecies.baseStats[i];
+			deltas.baseStats[i] = statChange;
+		}
+		deltas.types = [];
+		if (evo.types[0] !== prevoSpecies.types[0]) deltas.types[0] = evo.types[0];
+		if (evo.types[1] !== prevoSpecies.types[1]) {
+			deltas.types[1] = evo.types[1] || evo.types[0];
+		}
```

Comment:
```suggestion
		for (const [i, type] of evo.types) {
			if (type !== prevoSpecies.types[i]) deltas.types.push(type);
		}
		if (evo.types.length < prevoSpecies.types.length) deltas.types.push(evo.types[0]);
```

this should work but it is also untested so make sure to test this before trusting me

Reply:
imma fix it if it fails